### PR TITLE
Add Event_to_sketch in api

### PR DESCRIPTION
**Pull request by deralexxx**
https://github.com/google/timesketch/issues/691

Not yet ready to merge, maybe @berggren has opinion on the PR

e.g. do we need a unit test as well?

### CODE REVIEW

**Comment by deralexxx** [comment]
Ok there is still an issue seems the ID returned is not the id from the event. Not sure why

**Comment by deralexxx** [comment]
So the return json is:

```json
{
  "meta": {}, 
  "objects": [
    {
      "color": "7C79F3", 
      "created_at": "Tue, 12 Jun 2018 08:27:56 -0000", 
      "deleted": null, 
      "description": "internal timeline for user-created events", 
      "id": 18, 
      "name": "sketch specific timeline", 
      "searchindex": {
        "created_at": "Tue, 12 Jun 2018 08:27:52 -0000", 
        "deleted": null, 
        "description": "internal timeline for user-created events", 
        "id": 17, 
        "index_name": "cd0f4c15c00417d44c7e47414598a23b", 
        "name": "sketch specific timeline", 
        "status": [
          {
            "created_at": "Wed, 31 Oct 2018 06:30:41 -0000", 
            "id": 145, 
            "status": "ready", 
            "updated_at": "Wed, 31 Oct 2018 06:30:41 -0000"
          }
        ], 
        "updated_at": "Tue, 12 Jun 2018 08:27:52 -0000"
      }, 
      "status": [
        {
          "created_at": "Tue, 12 Jun 2018 11:03:41 -0000", 
          "id": 17, 
          "status": "new", 
          "updated_at": "Tue, 12 Jun 2018 11:03:41 -0000"
        }
      ], 
      "updated_at": "Tue, 12 Jun 2018 08:27:56 -0000"
    }
  ]
}

```

**Comment by deralexxx** [comment]
IMHO ready to merge

File: api_client/python/timesketch_api_client/client.py [hunk 1]
```diff
@@ -512,6 +512,27 @@ def search_by_label(self, label_name):
         }
         return self.explore(query_dsl=json.dumps({'query': query}))
 
+    def add_event(self, message, timestamp, timestamp_desc):
+        """Adds an event to the sketch specific timeline.
+
+        Args:
+            message: Array of JSON objects representing events.
+            timestamp: Timestamp of the event.
```
Comment (berggren) [inline api_client/python/timesketch_api_client/client.py]:
```suggestion
            timestamp: Micro seconds since 1970-01-01 00:00:00.
```

**Comment by deralexxx** [comment]
@berggren you need to make a author consent as far as I can see.

**Comment by berggren** [comment]
CLS bot is broken. deralexxx has signed the CLA.

**Pull request merged**